Assets/UI/Panels/citypanel.lua

```diff
@@ -1,3 +1,9 @@
+-- ===========================================================================
```

Comment:
I will comment below on any changes that are NOT simply comment lines for code clarity